source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+
+*Note*: The command is expected to take about 3-4 minutes to run.
+
+.. command-block::
+
+   qiime deblur denoise-16S \
+     --i-demultiplexed-seqs ./quality_filtered_seqs.qza \
+     --p-trim-length 150 \
+     --p-sample-stats \
+     --o-table ./deblur_table.qza \
+     --o-representative-sequences ./deblur_rep_set.qza \
+     --o-stats ./deblur_stats.qza
+
+We can also review the deblur stats using the ``qiime deblur visualize-stats`` command.
+
+.. command-block::
+
+    qiime deblur visualize-stats \
+      --i-deblur-stats ./deblur_stats.qza  \
+      --o-visualization ./deblur_stats.qzv
+
+Feature Table Summary
+---------------------
+
+After we finish denoising the data, we can check the quality filtering results. We can use two commands to explore the sequence data. First, we’ll look at the summary of the feature table. This will provide us with the counts associated with each sequence and each feature, as well as a histogram of the features.
+
+.. command-block::
+
+   qiime feature-table summarize \
+     --i-table ./deblur_table.qza \
+     --o-visualization ./deblur_table.qzv
+
+.. question::
+
+   1. How many features remain after denoising?
+   2. Which sample has the fewest sequences? How many does it have?
+   3. Which sample has the most? How many sequences does that sample have?
+   4. If we chose to filter the data to retain only samples with 2500 sequences, how many samples would we lose?
+   5. Which features are observed in at least 47 samples?
+
+Generating a Phylogenetic Tree for Diversity Analysis
+=====================================================
+
+QIIME 2 analysis allows the use of phylogenetic trees for both diversity metrics such as PD whole tree and UniFrac distance as well as feature-based analyses in Gneiss. The tree provides an inherent structure to the data, allowing us to consider an evolutionary relationship between organisms.
+
+QIIME 2 offers several ways to construct a phylogenetic tree. For this tutorial, we’re going to use a fragment insertion tree using the ``fragment-insertion`` plugin. The authors of the fragment insertion plugin suggest that it can outperform traditional alignment based methods based on short illumina reads by alignment against a reference tree built out of larger sequences. Our command, ``qiime fragment-insertion sepp`` will take the representative sequences (a ``FeatureData[Sequence]`` object) we generated during deblurring and return a phylogenetic tree where the sequences have been inserted into the greengenes 13_8 99% identity reference tree backbone.
```

Comment:
```suggestion
QIIME 2 offers several ways to construct a phylogenetic tree. For this tutorial, we’re going to use a fragment insertion tree using the ``fragment-insertion`` plugin. The authors of the fragment insertion plugin suggest that it can outperform traditional alignment based methods based on short Illumina reads by alignment against a reference tree built out of larger sequences. Our command, ``qiime fragment-insertion sepp`` will take the representative sequences (a ``FeatureData[Sequence]`` Artifact) we generated during deblurring and return a phylogenetic tree where the sequences have been inserted into the Greengenes 13_8 99% identity reference tree backbone.
```